GoogleSignIn/Tests/Unit/GIDSignInInternalOptionsTest.m

```diff
@@ -29,6 +29,32 @@ @interface GIDSignInInternalOptionsTest : XCTestCase
 
 @implementation GIDSignInInternalOptionsTest
 
+- (void)testDefaultOptionsForVerificationFlow {
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+  id configuration = OCMStrictClassMock([GIDConfiguration class]);
```

Comment:
I don't think you need to mock anything here. You can just create instances of these types.

Additionally, we are trying to refrain from mocking in general.

Reply:
Done.